omnigibson/object_states/slicer_active.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from omnigibson.macros import create_module_macros
+from omnigibson.object_states.object_state_base import AbsoluteObjectState
+from omnigibson.object_states.aabb import AABB
+from omnigibson.object_states.contact_bodies import ContactBodies
+from omnigibson.object_states.update_state_mixin import UpdateStateMixin
+import omnigibson as og
+
+
+# Create settings for this module
+m = create_module_macros(module_path=__file__)
+
+class SlicerActive(AbsoluteObjectState, UpdateStateMixin):
+    @classmethod
+    def get_dependencies(cls):
+        deps = super().get_dependencies()
+        deps.add(AABB)
+        deps.add(ContactBodies)
+        return deps
+
+    def __init__(self, obj):
+        super(SlicerActive, self).__init__(obj)
+
+        self._previously_touching_sliceable = False
+        self.value = True
+        self.delay_counter = 0
+        self.delay_steps = 7
```

Comment:
Don't make this an instance member, make it a macro/constant. Right after line 11 add m.REACTIVATION_DELAY = 7 etc.